### ISSUE TITLE
Incorporate new raster widget into production site

### ISSUE THREAD

**Issue by danrademacher** [stamen/ecoengine#185]
Once #184 is done, it needs to be pulled into production site and added to Explore UI (design and build time)

**Comment by syntagmatic** [comment]
And Compare UI

**Comment by syntagmatic** [comment]
First pass at integration here http://stamen.github.io/ecoengine/explore/

<img width="821" alt="screen shot 2016-03-09 at 4 37 35 pm" src="https://cloud.githubusercontent.com/assets/156229/13655559/1cd68fae-e615-11e5-89a5-ef3ead1cb1b8.png">

**Comment by syntagmatic** [comment]
Created a separate issue for Compare
